cmd/cronosd/cmd/root.go

```diff
@@ -146,6 +147,7 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 func addModuleInitFlags(startCmd *cobra.Command) {
 	crisis.AddModuleInitFlags(startCmd)
 	cronos.AddModuleInitFlags(startCmd)
+	startCmd.Flags().String(cronosclient.FlagStreamers, "", "Enable streamers, only file streamer is supported right now")
```

Comment:
nitty picky but would it be better to support enum type there?

maybe ok for the backporting version

Reply:
There'll be a better configuration system for streamers in another PR in cosmos-sdk.
For this version I think we just need to generate the stream files, then convert to version db with some offline script.